content/execution_model.tex

```diff
@@ -6,7 +6,7 @@
 start by calling the initialization routine  \FUNC{shmem\_init}
 \footnote{\textbf{start\_pes} has been deprecated as of Specification 1.2}
 before using any of the other \openshmem library routines.  An \openshmem
-program finishes execution by returning from the main routine or when any PE
+program finishes execution by returning from the main routine, or when any PE
```

Comment:
This comma seems unnecessary, but I think the desire to add it comes from the not-exactly-parallel construction of the two "or" clauses.

How about...?
> An OpenSHMEM program finishes execution when all PEs return from the `main` routine or any PE calls `shmem_global_exit`.